com.unity.render-pipelines.core/Editor/Volume/VolumeComponentListEditor.cs

```diff
@@ -97,15 +102,7 @@ public void Init(VolumeProfile asset, SerializedObject serializedObject)
             m_EditorTypes = new Dictionary<Type, Type>();
             m_Editors = new List<VolumeComponentEditor>();
 
-            // Gets the list of all available component editors
-            var editorTypes = CoreUtils.GetAllTypesDerivedFrom<VolumeComponentEditor>()
-                .Where(
-                    t => t.IsDefined(typeof(VolumeComponentEditorAttribute), false)
-                    && !t.IsAbstract
-                );
-
-            // Map them to their corresponding component type
-            foreach (var editorType in editorTypes)
+            foreach (var editorType in TypeCache.GetTypesWithAttribute<VolumeComponentEditorAttribute>())
```

Comment:
You've removed the !IsAbstract requirement, do you think this attribute is never given to abstract classes?

Reply:
I think it should never be given to abstract classes, but you are right better be safe. I have changed the VolumeComponentEditor for the official CustomEditor for Unity, in this way users are more used to that pattern and besides this we have all the checks that they are performing and the editor creation from trunk.

Reply:
Thanks. Now that this is a much bigger change, do you have any context for what the original motivation behind using the custom VolumeComponentEditorAttribute instead of CustomEditorAttribute? 

This should also get a full QA pass to avoid regressions.

Reply:
Basically, everything on Volumes is custom, because while ago when this was developed we didn't have this classes